include/envoy/router/router.h

```diff
@@ -789,6 +789,12 @@ class RouteEntry : public ResponseEntry {
    */
   virtual std::chrono::milliseconds timeout() const PURE;
 
+  /**
+   * @return bool measure timeout() from when the request is started, instead of when it is
```

Comment:
Can you be more specific about what "started" means? Does this mean after a connection is received from the connection pool? Or before a connection is requested from the pool?